akka-persistence-testkit/src/main/scala/akka/persistence/testkit/ChangePersisted.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2022 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.testkit
+
+/** Common superclass denoting a persistence effect from an EventSourcedBehavior or a DurableStateBehavior.
+ *
+ *  From an EventSourcedBehavior, a ChangePersisted will be either an EventPersisted (corresponding to an
+ *  Effect.persist or Effect.persistAll) or a StatePersisted (corresponding to a snapshot being persisted).
```

Comment:
the API does not seem like it quite matches this `persistAll` would be multiple `ChangePersisted` no?

Reply:
This whole API will be removed shortly.